src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/VisualBasicCompilerExtensions.projitems

```diff
@@ -13,5 +13,6 @@
     <Compile Include="$(MSBuildThisFileDirectory)Extensions\SyntaxTreeExtensions_SharedWithCodeStyle.vb" />
     <Compile Include="$(MSBuildThisFileDirectory)Extensions\SyntaxTriviaExtensions.vb" />
     <Compile Include="$(MSBuildThisFileDirectory)Helpers\RemoveUnnecessaryImports\VisualBasicUnnecessaryImportsProvider.vb" />
+    <Compile Include="$(MSBuildThisFileDirectory)Services\SyntaxFacts\VisualBasicSyntaxKindsService.vb" />
```

Comment:
Maybe name the folder `LanguageServices` for consistency with other projects.

Reply:
So this was intentional.  At this level it is *not* a LanguageService :)  The LanguageService stuff is introduced higher up.  If we can push language services down later to this level, then we should rename this folder.

Reply:
Ah, yes that is correct.